### PR TITLE
Updating video sample to allow for a seamless switch between pdf/video viewer

### PR DESCRIPTION
**Pull request by willywwang**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -146,7 +91,81 @@ const App = () => {
 
   async function onFileChange(event) {
     const url = URL.createObjectURL(event.target.files[0]);
-    wvLoadVideo(url);
+
+    // Can also detect by specific video file types (ie. mp4, ogg, etc.)
+    if (event.target.files[0].type.includes('video')) {
+      initializeWebViewerSettings(wvInstance);
+      wvLoadVideo(url);
+      setTimeout(() => {
```
Comment (willywwang) [inline src/App.js:99]: Notes panel wouldn't open unless a small time out was added, probably due to webviewer instance resolving video.

**Comment by willywwang** [comment]
Changes are dependent on this PR: https://github.com/XodoDocs/WebViewer-video/pull/40

File: src/App.js (hunk 1 above)
Comment (khein-pdftron) [reply src/App.js:99]: Why does it need the timeout on the first load but it does when you switch?

Comment (willywwang) [reply src/App.js:99]: not 100% sure here, I was trying to do some debugging on the lifecycle of webviewer, but went down the rabbit hole for a few hours :)

**Pull request merged**